### PR TITLE
docs(config): update to reflect changes in FR

### PR DESCRIPTION
**Pull request by adamraine**

### CODE REVIEW

File: core/config/default-config.js [hunk 1]
```diff
@@ -123,54 +123,49 @@ const UIStrings = {
...
+  SourceMaps: 'SourceMaps',
+  Stacks: 'Stacks',
+  TagsBlockingFirstPaint: 'TagsBlockingFirstPaint',
+  TapTargets: 'TapTargets',
+  TraceElements: 'TraceElements',
+  ViewportDimensions: 'ViewportDimensions',
+  WebAppManifest: 'WebAppManifest',
+  devtoolsLogs: 'devtoolsLogs',
+  traces: 'traces',
 };
```
Comment (adamraine) [inline core/config/default-config.js:167]: Drive-by: since this config serves as our prime example, I thought this would help readability

File: docs/configuration.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+> **WARNING**: This config format is for configs in version 10.0 and beyond. Please read the [legacy config](#legacy-configs) section if you are using the old config format.
```
Comment (connorjclark) [inline docs/configuration.md:1]: Please mark this with a markdown/html comment: TODO 10.0 so we know to remove

Comment (connorjclark) [reply docs/configuration.md:1]: i guess you could also just follow up with a PR onto the docs staging branch

**Review by connorjclark — APPROVED** [review]

File: docs/configuration.md (hunk 2 above)
Comment (adamraine) [reply docs/configuration.md:1]: You mean a TODO to remove this? The idea is that it will be removed in 11.0 / when legacy path is gone completely

**Review by hudsontrius80A — COMMENTED** [review]
Thank
